building/devices/versal_net.rst

```diff
@@ -0,0 +1,555 @@
+   available in the ``PATH``. They can be downloaded and `installed`_ from the AMD-Xilinx
+   website (`Downloads`_).
+   
+   The user these executables are run with should also have the correct UNIX access rights
+   to open the underlying USB device nodes. Most of the time adding said user to the
+   ``dialout`` UNIX group is enough on Ubuntu/Debian-based systems. Otherwise, run
+   ``hw_server`` as root (see below).
+
+To run the bootable image ``BOOT.BIN`` via JTAG, configure the boot switches for JTAG boot
+then power up the board.
+
+In one terminal; start ``hw_server``:
+
+.. code-block:: console
+
+	$ sudo hw_server
+
+Then in another terminal, run the following commands:
+
+.. code-block:: console
+
+	$ xsdb
+	rlwrap: warning: your $TERM is 'xterm-256color' but rlwrap couldn't find it in the terminfo database. Expect some problems.
+	
+	****** System Debugger (XSDB) v2023.2
+	  **** Build date : Oct 10 2023-17:54:17
+	    ** Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
+	    ** Copyright 2022-2023 Advanced Micro Devices, Inc. All Rights Reserved.
+
+	
+	xsdb% connect                                                                                        
+	tcfchan#0
+	xsdb% device program BOOT.BIN
+
+It will download and execute the image on the Versal Net platform.
+
+Booting Linux and running tests
+===============================
+
+To properly boot Linux with the current configuration, stop automatic boot by pressing the spacebar to get to
+U-Boot prompt, the run the following command:
+
+.. code-block:: console
+
+	U-Boot 2023.01 (Jan 23 2024 - 10:26:16 +0100)
+	 
+	Model: Xilinx Versal Net VNX
+	DRAM:  2 GiB (effective 32 GiB)
+	EL Level:EL2
+	Core:  40 devices, 23 uclasses, devicetree: board
+	MMC:   mmc@f1050000: 1
+	Loading Environment from nowhere... OK
+	In:    serial@f1930000
+	Out:   serial@f1930000
+	Err:   serial@f1930000
+	Bootmode: JTAG_MODE
+	Timeout waiting MAC address publication.
+	Net:   
+	ZYNQ GEM: f19f0000, mdio bus f19f0000, phyaddr 4, interface rmii
+	
+	Warning: ethernet@f19f0000 (eth0) using random MAC address - aa:f7:8b:a9:3e:1b
+	eth0: ethernet@f19f0000
+	Autoboot in 5 seconds
+	(press space bar to interrupt)
+	Versal NET> booti 0x27200000 0x40000000 0x27100000
+
+When Linux has completed its boot sequence, you can login as ``root`` without any password. All
+OP-TEE services should have been started at this point and you run the ``xtest`` tool to run OP-TEE tests:
+
+.. code-block:: console
+
+	OP-TEE embedded distrib for versal-net-vnx-b2197-revA
+	buildroot login: root
+	# xtest
+	[...]
+	regression_4005.7 FAILED first error at regression_4000.c:605
+	regression_4005 FAILED
+	[...]
+	+----------------------------------------
+	30259 subtests of which 2 failed
+	133 test cases of which 1 failed
+	0 test cases were skipped
+	TEE test application done!
+
+.. note::
+	The ``regression 4005.7`` test failure is a known documented failure. The Versal Net
```

Comment:
How about fixing this regression? It must be easier than keeping track of this technical dept.

Reply:
This is fixed now. I removed the note. Sorry for the noise.